sections/dashboard/FuturesPositionsTable/FuturesPositionsTable.tsx

```diff
@@ -60,6 +59,15 @@ const LegacyLink = () => {
 	);
 };
 
+// - Market
+// - Side
+// - Size
+// - Average Entry
+// - Market Margin
+// - Leverage
+// - Liquidation Price
+// - PnL
+
 const FuturesPositionsTable: FC<FuturesPositionTableProps> = ({
```

Comment:
I think we said about just using the one table across both dashboard and market page, so can remove this now and configure the other one to not include the action buttons when used on dashboard.

Reply:
I would say don't worry about the order for now. But let's do this in a different PR anyway.